tsconfig.json

```diff
@@ -6,7 +6,15 @@
 		"target": "es6",
 		"lib": [
 			"es2015"
-		]
+		],
```

Comment:
any reason to include this since we are compiling to es6 (es2015) already ?

Reply:
oh realized it was not part of your change but shoudn't be any reason AFAIK to include this.

---
In reply to: [253675090](https://github.com/Microsoft/vscode-go/pull/2294#discussion_r253675090) [](ancestors = 253675090)

Reply:
What do you think @ramya-rao-a?

I don't work with typescript on a regular basis (unfortunately) so my knowledge of project configuration is next to none

Reply:
FYI lib is just so TS works with features of the specified lib ( it doesnt affect the generated JS ), so in general if by removing it, the code compiles, should be fine. You can leave it as is now if you prefer.